Yehor/reactNetflix/src/components/detailedMovie/DetailedMovie.tsx

```diff
@@ -7,9 +7,15 @@ interface IMovieProps {
 }
 
 function DetailedMovie({ movie }: IMovieProps) {
+
+  const defimg = (e: React.SyntheticEvent<HTMLImageElement>) => {
+    const target = e.target as HTMLImageElement;
+    target.src = "https://media.comicbook.com/files/img/default-movie.png";
+  }
+
   return (
     <div className="dmovie">
-      <img className="dmovie_img" src={movie.poster_path} alt={movie.title} />
+      <img src={movie.poster_path} alt={movie.title} onError={defimg} className="dmovie_img"/>
```

Comment:
Would be grate if you've made your reusable ImageFallback component with current logic inside. So, you could reuse this component, when needed